torch/_functorch/vmap.py

```diff
@@ -458,22 +560,6 @@ def _get_chunk_flat_args(flat_args_, flat_in_dims_, chunks_):
         chunks_flat_args = zip(*flat_args_chunks)
         return chunks_flat_args
 
-    def _flatten_chunks_output(chunks_output_):
-        # chunks_output is a list of chunked outputs
-        # flatten chunked outputs:
-        flat_chunks_output = []
-        arg_spec_list = []
-        for output in chunks_output_:
```

Comment:
Can you add a note into the chunk_vmap docstring to use vmap with chunk_size instead?